README.md

```diff
@@ -1,50 +1,53 @@
-# Project Name
-> This project is a portfolio website which is created using pure Html5 and Css3.
-> Me and my other two partners have worked together on form section in this project.
-> This project has both features its mobile friendly as well as desktop too.
-> This portfolio project is now hosted and the link is given in the deployement section.
-## Built With
-- Html
-- Css
-
-## Getting Started
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-Git must be installed on you machine.
-
-### Setup
-Clone this repository by using the command line :
-`git clone https://github.com/tanveerisonline/personal-portfolio-website.git
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-https://tanveerisonline.github.io/portfolio_website/
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@tanveerisonline](https://github.com/tanveerisonline)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
+# Project Name
+
+> This project is a portfolio website which is created using pure Html5 and Css3.
+> Me and my other two partners have worked together on form section in this project.
+> This project has both features its mobile friendly as well as desktop too.
+
+## Built With
+
+- Html
+- Css
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Git must be installed on you machine.
+
+### Setup
+
+Clone this repository by using the command line :
+`git clone https://github.com/tanveerisonline/personal-portfolio-website.git`
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
### OPTIONAL
- [ ] I recommend you fill many sections as you can. But if you won't fill sections better to delete them. It is important to make a professional readme file.